nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -41,6 +41,12 @@ let
     in
       "${if pkgs.stdenv.buildPlatform == pkgs.stdenv.hostPlatform then Caddyfile-formatted else Caddyfile}/Caddyfile";
 
+  etcConfigFile = "caddy/Caddyfile";
+
+  configPath = if cfg.enableReload
+    then "/etc/${etcConfigFile}"
+    else configFile;
+
```

Comment:
I just want to mention it might lead to confusion using `enableReload` with something that is not a Caddyfile such as json, using `configFile` and `adapter`, especially considering that changing the `adapter` will trigger a restart (as changing the adapter alters the systemd commands).

But honestly, that's alright.
Really just wanted to mention it for the record :)